Pear/Controller/LoginViewController.swift

```diff
@@ -132,10 +134,21 @@ extension LoginViewController: GIDSignInDelegate {
               // Only allow users with Cornell emails or internal users (Pear email) to log in
               email.contains("@cornell.edu") || email == "cornellpearapp@gmail.com" else {
             GIDSignIn.sharedInstance().signOut()
+            do {
+                try Auth.auth().signOut()
+            } catch let signOutError as NSError {
```

Comment:
Catch clauses implicitly give you an `error`. You can just `} catch {` and print `error` in the body.